internal/command/postgres/config_update.go

```diff
@@ -311,20 +316,20 @@ func resolveConfigChanges(ctx context.Context, app *fly.AppCompact, manager stri
 		if !autoConfirm {
 			const msg = "Are you sure you want to apply these changes?"
 
-			switch confirmed, err := prompt.Confirmf(ctx, msg); {
+			switch shouldApply, err = prompt.Confirmf(ctx, msg); {
 			case err == nil:
-				if !confirmed {
-					return false, nil, nil
+				if !shouldApply {
+					return false, false, nil, nil
```

Comment:
Can we just return `fmt.Errorf("cancelled")`? Looks like other early returns use this already.

This will simplify the return type - it's pretty hard to understand `(bool, bool, map[string]string, error) ` without reading the code.

Reply:
Nice, wondered if we could use the error to short-circuit that path for the same reasons as you. Fixed.